aws/ssm/plugin.go

```diff
@@ -30,11 +31,33 @@ func StartSession(ctx context.Context, session interface{}, target ssm.StartSess
 		string(targetRaw),
 		endpointUrl,
 	}
+	ctx = context.Background() // Ignore signal cancellations on the context
+
 	cmd := exec.CommandContext(ctx, process, args...)
 	cmd.Stderr = os.Stderr
 	cmd.Stdout = os.Stdout
 	cmd.Stdin = os.Stdin
-	return cmd.Run()
+	if err := cmd.Start(); err != nil {
+		return err
+	}
+
+	done := make(chan any)
+	defer close(done)
+	forwardSignals(done, cmd.Process)
+	return cmd.Wait()
+}
+
+func forwardSignals(done chan any, process *os.Process) {
+	ch := make(chan os.Signal, 1)
+	signal.Notify(ch)
+	go func() {
+		select {
+		case <-done:
```

Comment:
Does this mean that the way to exit the ssh session is to "signoff" of the machine by typing `exit`?

Reply:
Correct, you can either type `exit` or press `Cmd+D` or `Ctrl+D`.
This makes the `nullstone ssh` command more consistent with the `ssh` command.